ui/perfherder/compare/TableAverage.jsx

```diff
@@ -14,7 +15,15 @@ const TableAverage = ({ value, stddev, stddevpct, replicates, app }) => {
       .map((value) => formatNumber(value))
       .join(' ')} > ${formatNumber(displayNumber(stddev))} = ${formatNumber(
       displayNumber(stddevpct),
-    )}% standard deviation)`;
+    )}% standard deviation`;
+
+    if (tooltipText.length > tooltipMaxLength) {
+      tooltipText = `${tooltipText.slice(0, Math.floor(tooltipMaxLength / 2))}
+        ...(use JSON download button to see more)...
+        ${tooltipText.slice(
+          tooltipText.length - Math.floor(tooltipMaxLength / 2),
+        )}`;
+    }
```

Comment:
```
let tooltipText;
  let replicatesStr;
  if (replicates.length > 1) {
    replicatesStr = replicates.map((value) => formatNumber(value)).join(' ');

    const replicatesMaxLength = 200;
    if (replicatesStr.length > replicatesMaxLength) {
      tooltipText = (
        <>
          {`Runs: < ${replicatesStr.slice(
            0,
            Math.floor(replicatesMaxLength / 2),
          )}`}
          ...
          {`${replicatesStr.slice( replicatesStr.length - Math.floor(replicatesMaxLength / 2), )} > `}
          {`${formatNumber(displayNumber(stddev))} = ${formatNumber(displayNumber(stddevpct),)}%`}
          standard deviation
          <br />
          (use JSON download button to see more)
        </>
      );
    } else {
      tooltipText = (
        <>
          {`Runs: < ${replicatesStr} > ${formatNumber(
            displayNumber(stddev),
          )} = ${formatNumber(displayNumber(stddevpct))}% standard deviation`}
        </>
      );
    }
  } else if...
```

Reply:
![image](https://github.com/mozilla/treeherder/assets/47977085/a2573134-f9b6-4221-98a7-a0df621b323e)

Reply:
reference: https://github.com/mozilla/treeherder/blob/ba44294abecbbc203e1327f8f9ad59114bf77a50/ui/perfherder/alerts/AlertTableRow.jsx#L367

Reply:
Ah nice! I had no idea we could do this.